### PR TITLE
docs: Migrate to Docusaurus v2.x

### PR DESCRIPTION
**Pull request by jaroslav-kubicek**
Done according to https://v2.docusaurus.io/docs/migrating-from-v1-to-v2/

### CODE REVIEW

**Comment by jaroslav-kubicek** [comment]
Do we need to change something for deployment? `publish-gh-pages` was changed to deploy

**Comment by mrtnzlml** [comment]
> Do we need to change something for deployment? `publish-gh-pages` was changed to deploy

I don't know. What is `publish-gh-pages`?

**Comment by jaroslav-kubicek** [comment]
That's a good question, I haven't found it anywhere in the repo and docusaurus docs say I should replace it by deploy

File: src/docs/docusaurus.config.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+module.exports = {
+  url: 'https://adeira.dev/',
+  baseUrl: '/',
+  favicon: 'img/favicons/favicon.ico',
+  title: 'adeira/universe', // Title for your website.
+  tagline: 'Universe of libs to write better code.',
+  // Add custom scripts here that would be placed in <script> tags.
+  scripts: ['https://buttons.github.io/buttons.js'],
+  // Used for publishing and more
+  projectName: 'universe',
```
Comment (jaroslav-kubicek) [inline src/docs/docusaurus.config.js:12]: this should be `adeira.dev`

**Pull request merged**